README.md

```diff
@@ -31,3 +35,14 @@ simpletrend.pyx
 summation.pyx
 validmask.pyx
 ```
+
+## Dev Setup
+
+Build a container, set an alias to let you run tests using your host machine's
+working copy, then run the test suite:
+
+```
+$ time docker build . -t mt --no-cache
+$ alias rmt="docker run --rm -it -v ${HOME}/src/multitemporal/:/multitemporal"
```

Comment:
This doesn't work in general because of the assumption of $HOME/src. But I think the mount isn't needed and could be left out.

Reply:
The text above this says "Build a container, set an alias to let you run tests using your host machine's working copy, then run the test suite:"

Do you think that makes it clear enough to let it go?

Reply:
resolved by additional commit.